.circleci/config.yml

```diff
@@ -0,0 +1,104 @@
+version: 2.1
+
+parameters:
+  deploy_bucket:
+    type: string
+    default: "pip.astronomer.io"
+
+workflows:
+  build_and_deploy:
+    jobs:
+      - build:
+          filters:  # required since `deploy` has tag filters AND requires `build`
+            tags:
+              only: /.*/
+      - publish:
+          context: "Pip Deploy"
+          requires: [build]
+          # Run this stage only for version-number tagged builds
+          filters:
+            tags:
+              only: /^v\d+[.\d]+.*/
+            branches:
+              ignore: /.*/
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.7-buster
+    environment:
+      PIP_DISABLE_PIP_VERSION_CHECK: 'True'
+      PYTEST_ADDOPTS: "--color=yes"
+
+    steps:
+      - checkout
+      - &restore_venv
+        restore_cache:
+          keys:
+            - deps-{{ .Branch }}-{{ checksum "setup.py" }}
+            - deps-master-{{ checksum "setup.py" }}
+      - run:
+          name: Install test deps
+          command: |
+            # Use a virtual env to encapsulate everything in one folder for caching
+            python -m venv .venv
+            . .venv/bin/activate
+            pip install -e '.[test]'
+      - save_cache:
+          key: deps-{{ .Branch }}-{{ checksum "setup.py" }}
+          paths:
+            - ".venv"
+            - "~/.cache"
+      - run:
+          name: run_tests
```

Comment:
what about to run flake8 as it was in drone?

Reply:
```
    commands:
    - pip install flake8 'flake8-import-order>=0.18'
    - flake8
```

Reply:
Oh yes, good spot.

(I should add that to my other new plugin too.)

Reply:
Added